Wikipedia/Code/WMFArticleListTableViewCell.m

```diff
@@ -52,6 +52,8 @@ - (void)awakeFromNib {
     [super awakeFromNib];
     [self configureImageViewWithPlaceholder];
     [self wmf_makeCellDividerBeEdgeToEdge];
+    self.titleLabel.textAlignment       = NSTextAlignmentNatural;
```

Comment:
I tried but it didn't seem to work... was going to try to track down why but I forgot.